docs/vscode-extension/envchecker-help.md

```diff
@@ -80,6 +80,14 @@ It might be caused by timeout issue (longer than 3 minutes), the process to inst
 ### Mitigation
 Please refer to [the guide](#how-to-install-net-sdk) to install `.NET SDK` manually.
 
+## DotnetNotSupportTargetVersion
+### Notification Message
+> NETSDK1045: The current .NET SDK does not support 'newer version' as a target.
+
+### Mitigation
+Please refer to [the guide](https://docs.microsoft.com/dotnet/core/tools/sdk-errors/netsdk1045#globaljson-file) to check your global.json file in the root folder in your project and up the directory chain to the root of the volume, since it can be anywhere in the folder structure. If it contains an SDK version, delete the sdk node and all its children, or update it to the desired newer .NET Core version(`.NET 5` or `.NET 3.1` ).
```

Comment:
nit: 
* global.json => `global.json`
* `If it contains an SDK version` => `If a SDK version is defined in global.json`